benchmarks/encoder/utils.py

```diff
@@ -47,7 +60,7 @@ def gen_traces(config):
             span_name = random.choice(span_names)
             resource = random.choice(resources)
             service = random.choice(services)
-            with Span(None, span_name, resource=resource, service=service, parent_id=parent_id) as span:
+            with Span(None, span_name, resource=resource, service=service, parent_id=parent_id, context=Context()) as span:
```

Comment:
Why are we adding context here?

Reply:
Added comment. Reason is to support the `dd_origin` changes before 0.49.